src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random randomNumber = new Random();
```

Comment:
You don't need it here, you need to assign values to the fields from the construct parameters.